src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const males = century
+    ? people.filter(person => (
+      person.sex === 'm'
+      && Math.ceil(person.died / 100) === century
+    ))
+    : people.filter(person => (
+      person.sex === 'm'
+    ));
```

Comment:
To avoid repetition, lets do it like this: 
```suggestion
const males = people.filter(person => person.sex === 'm' && (!century || Math.ceil(person.died / 100) === century));
```